pkg/reconciler/broker/controller.go

```diff
@@ -91,7 +92,12 @@ func newController(ctx context.Context, cmw configmap.Watcher, drs *dataresidenc
 		dataresidencyStore: drs,
 	}
 
-	impl := brokerreconciler.NewImpl(ctx, r, brokerv1beta1.BrokerClass)
+	impl := brokerreconciler.NewImpl(ctx, r, brokerv1beta1.BrokerClass,
+		func(impl *controller.Impl) controller.Options {
+			return controller.Options{
+				ConfigStore: brds,
```

Comment:
Interesting that `brds` is set as `ConfigStore` here, while `drs` is passed in as `dataresidencyStore` field above. Is this due to a limitation in the reconciler code?

Reply:
I'm not sure. The data residency store is loaded manually during broker reconciling, so it is attached to the reconciler itself. I didn't need to load the delivery store manually so I used this option instead.